.github/workflows/docker_image.yml

```diff
@@ -0,0 +1,127 @@
+name: Docker build and Push
+
+# Trigger the workflow on pull request
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    
+    # Checkout the repository to the GitHub Actions runner
+    - uses: actions/checkout@v3
+
+
+    # Set up Python
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    - name: Get branch name
+      id: branch_name
+      run: echo "::set-output name=branch::${GITHUB_HEAD_REF}"
+      
+    # Install yaml
+    - name: Install yaml and oyaml
+      run: |
+          pip install pyyaml
+          pip install oyaml
+
+    # Install linkml
+    - name: Install LinkML
+      run: pip install linkml
+
+    # Check if the PR description "Organization Name" has a value and matches the one in the model card
+    - name: Check PR description
+      if: github.event_name == 'pull_request' && github.event.pull_request.merged != true
```

Comment:
This workflow is already designed for `on PR`. Probably, check for event_name is redundant. Since this check is done at the opening of a PR, can we simply replace `github.event.pull_request.merged != true` with `github.event.pull_request.opened == true`

Reply:
@hvgazula I double-checked and there is no option for .opened, I guess we will have to stick with .merged

Reply:
How about this `- if: ${{ github.event_name == 'pull_request' && github.event.action == 'opened' }}`

Reply:
Yes. That would work 👍